packages/consumption/test/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.test.ts

```diff
@@ -710,20 +874,20 @@ describe("ReadAttributeRequestItemProcessor", function () {
             const result = await processor.accept(requestItem, acceptParams, incomingRequest);
             expect(result).toBeInstanceOf(AttributeSuccessionAcceptResponseItem);
 
-            const successorOSIA = await consumptionController.attributes.getLocalAttribute((result as AttributeSuccessionAcceptResponseItem).successorId);
-            expect(successorOSIA).toBeDefined();
-            expect(successorOSIA!.shareInfo).toBeDefined();
-            expect(successorOSIA!.shareInfo!.peer.toString()).toStrictEqual(incomingRequest.peer.toString());
-            expect(successorOSIA?.shareInfo!.sourceAttribute).toStrictEqual(successorSourceAttribute.id);
-            expect(successorOSIA!.succeeds).toStrictEqual(predecessorOSRA.id);
+            const successorOwnSharedRelationshipAttribute = await consumptionController.attributes.getLocalAttribute((result as AttributeSuccessionAcceptResponseItem).successorId);
+            expect(successorOwnSharedRelationshipAttribute).toBeDefined();
+            expect(successorOwnSharedRelationshipAttribute!.shareInfo).toBeDefined();
+            expect(successorOwnSharedRelationshipAttribute!.shareInfo!.peer.toString()).toStrictEqual(incomingRequest.peer.toString());
+            expect(successorOwnSharedRelationshipAttribute?.shareInfo!.sourceAttribute).toStrictEqual(successorSourceAttribute.id);
+            expect(successorOwnSharedRelationshipAttribute!.succeeds).toStrictEqual(predecessorOwnSharedRelationshipAttribute.id);
 
-            const updatedPredecessorOSIA = await consumptionController.attributes.getLocalAttribute(predecessorOSRA.id);
-            expect(updatedPredecessorOSIA!.succeededBy).toStrictEqual(successorOSIA!.id);
+            const updatedPredecessorOwnSharedRelationshipAttribute = await consumptionController.attributes.getLocalAttribute(predecessorOwnSharedRelationshipAttribute.id);
+            expect(updatedPredecessorOwnSharedRelationshipAttribute!.succeededBy).toStrictEqual(successorOwnSharedRelationshipAttribute!.id);
         });
 
         test("accept with existing third party owned RelationshipAttribute whose predecessor was already shared", async function () {
```

Comment:
To be more precise, the test is about accepting with a "peer shared RelationshipAttribute" whose predecessor was already shared. A third party owned RelationshipAttribute is already a copy of a peer shared RelationshipAttribute and it should not be allowed to respond to a Request with such an Attribute.